src/library-authoring/data/apiHooks.ts

```diff
@@ -613,7 +613,7 @@ export const useContainer = (containerId?: string) => (
 /**
  * Use this mutation to update the fields of a container in a library
  */
-export const useUpdateContainer = (containerId: string) => {
+export const useUpdateContainer = (containerId: string, affectedParentContainerId?: string) => {
```

Comment:
Could you please just update the JSDoc comment about to explain what the new `affectedParentContainerId` parameter is doing exactly?